tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 # Minimum CMake required
-cmake_minimum_required(VERSION 3.5)
+if(WIN32)
+  cmake_minimum_required(VERSION 3.8)
```

Comment:
Why do we need to bump this dependency? is it the "-T" flag?

Reply:
I mean, why just windows, not also linux?

Reply:
3.8 is needed for the -T flag. The reasons for keeping the 3.5 version for non windows platforms are:
1) The cmake package (installed with "apt install") on Ubuntu 16.04 is version 3.5. Installing a newer version requires a manual download or adding another source to apt.
2) You only need the -T flag for windows.